packages/js/src/components/ImageSelectComponent.js

```diff
@@ -137,6 +160,8 @@ class ImageSelectComponent extends Component {
 				hasNewBadge={ this.props.hasNewBadge }
 				isDisabled={ this.props.isDisabled }
 				hasPremiumBadge={ this.props.hasPremiumBadge }
+				warnings={ this.state.warnings }
+				hasImageValidation={ this.props.hasImageValidation }
```

Comment:
Not sure why we pass this as a prop since `ImageSelect` doesn't use it?

Reply:
Oh yes, probably something I missed. Good catch 👍